neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Neo.SmartContract.Native.Tokens
+{
+    public sealed class GasToken : Nep5Token<Nep5AccountState>
+    {
+        public override string ServiceName => "Neo.Native.Tokens.GAS";
+        public override string Name => "GAS";
+        public override string Symbol => "gas";
+        public override int Decimals => 8;
+
+        internal GasToken()
```

Comment:
Can we include a variable for starting the Genesis with a certain amount of GAS?
This can be used for different things, one of them is:

* Initializing private-nets with an instantaneous amount of GAS available for tests;

Reply:
You can add two transfer, one of neo for claim, and the othhñer as you said

Reply:
Yes, sounds good. Just create an option to choose initial supplies.

It would also be interesting to initialize balance of address automatically in the genesis.